### PR TITLE
Docs: Describe how to use CI builds in projects (resolves #173)

### LINKED ISSUE #173: Add a doc/wiki describing how to use CI builds in projects
See https://github.com/DevExpress/DevExtreme.AspNet.Data/issues/85#issuecomment-277993703

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Comment by AlekseyMartynov** [comment]
answering https://github.com/DevExpress/DevExtreme.AspNet.Data/issues/173#issuecomment-358231339

> Why not just using-ci-builds.md? Wher else CI builds can be used? :)
>
> @San4es I don't know, for distributing :) I kept "in development" for now. Aleksey, how do you think?

It can be development or distributing or testing or random curiosity.

File: docs/using-ci-builds.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Using CI Builds
+
+Master branch build artifacts are stored [here](https://ci.appveyor.com/project/dxrobot/devextreme-aspnet-data/branch/master/artifacts). Download the `devextreme-aspnet-data-99.0.0-ci-NNN.tgz` file and use the instruction for the client-side script, or get `DevExtreme.AspNet.Data.99.0.0-ci-NNN.nupkg` and follow the one for the NuGet package.
```
Comment (AlekseyMartynov) [inline docs/using-ci-builds.md]: "or" is misleading.
If there's no special reason, you want to use matching versions of .dll and .js.
Both need to be updated.

File: docs/using-ci-builds.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Client-Side Script
+
+Unpack the `.tgz` archive to a temporary location. On Windows, you can use [7zip](http://www.7-zip.org/download.html) for that:
+
+```bash
+7zip x devextreme-aspnet-data-99.0.0-ci-NNN.tgz
+7zip x devextreme-aspnet-data-99.0.0-ci-NNN.tar
+```
+
+Navigate to the package/js folder in this location and copy the extracted JavaScript file to your project.
```
Comment (AlekseyMartynov) [inline docs/using-ci-builds.md]: "this location" is not clear for me.
Can we rephrase in terms of "extracted files" or something like this?

File: docs/using-ci-builds.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+For Node projects, you can install the archive as a package:
```
Comment (AlekseyMartynov) [inline docs/using-ci-builds.md]: "install the archive as a package" - I suggest linking https://docs.npmjs.com/cli/install

**Comment by San4es** [comment]
@AlekseyMartynov @RomanTsukanov 

Guys, maybe it would be good to add a link to Readme.md for this "How to" file to increase its discoverability?

**Comment by AlekseyMartynov** [comment]
@San4es 
Agree. But with a preamble explaining that this is only for workarounds and testing unreleased changes.

File: docs/using-ci-builds.md (hunk 2 above)
Comment (RomanTsukanov) [reply docs/using-ci-builds.md]: Well, it states "Unpack the archive to a **temporary location**" a couple of lines before, what can be unclear?

**Comment by RomanTsukanov** [comment]
@San4es @AlekseyMartynov done

File: docs/using-ci-builds.md (hunk 2 above)
Comment (AlekseyMartynov) [reply docs/using-ci-builds.md]: Maybe it's just me.

**Pull request merged**